### PR TITLE
[website] Make Comparison snippet data relevantly.

### PR DESCRIPTION
**Pull request by imjaypatel**
<b>Pad</b> : https://pad.odoo.com/p/r.UQAC5ngtaTJZKXnJ
<b>Task</b> : https://www.odoo.com/web#id=12111&view_type=form&model=project.task&menu_id=3942&action=327

### CODE REVIEW

File: addons/website/views/snippets.xml [hunk 1]
```diff
@@ -389,20 +389,20 @@
...
                   </div>
 
 
                   <ul class="list-group">
-                    <li class="list-group-item active">Battery: 12 hours</li>
-                    <li class="list-group-item">Screen: 2.8 inch</li>
-                    <li class="list-group-item">Weight: 1.2 ounces</li>
-                    <li class="list-group-item">Limited support</li>
+                    <li class="list-group-item active">Complete CRM for any size team</li>
+                    <li class="list-group-item">Get access all module</li>
```
Comment (bst-odoo) [inline addons/website/views/snippets.xml]: Typo : Get access to all modules

File: addons/website/views/snippets.xml [hunk 2]
```diff
@@ -414,24 +414,24 @@
...
                   </div>
 
 
                   <ul class="list-group">
-                    <li class="list-group-item active">Battery: 20 hours</li>
-                    <li class="list-group-item">Screen: 2.8 inch</li>
-                    <li class="list-group-item">Weight: 1.2 ounces</li>
-                    <li class="list-group-item">Unlimited support</li>
+                    <li class="list-group-item active">Unlimited CRM power and support</li>
+                    <li class="list-group-item">Get access all module + features</li>
```
Comment (bst-odoo) [inline addons/website/views/snippets.xml]: Typo : Get access to all modules and features

File: addons/website/views/snippets.xml (hunk 1 above)
Comment (jpa-odoo) [inline addons/website/views/snippets.xml]: ok,

File: addons/website/views/snippets.xml (hunk 2 above)
Comment (jpa-odoo) [inline addons/website/views/snippets.xml]: ok

**Pull request merged**